packages/ketcher-core/src/application/editor/Editor.ts

```diff
@@ -74,6 +78,22 @@ export class CoreEditor {
     return editor;
   }
 
+  private handleHotKeyEvents(event) {
+    const keySettings = hotkeysConfiguration;
+    const hotKeys = initHotKeys(keySettings);
+    const shortcutKey = keyNorm.lookup(hotKeys, event);
+
+    if (keySettings[shortcutKey] && keySettings[shortcutKey].handler) {
+      keySettings[shortcutKey].handler(this);
+      event.preventDefault();
+    }
+  }
+
+  setupHotKeysEvents() {
+    this.hotKeyEventHandler = (event) => this.handleHotKeyEvents(event);
+    document.addEventListener('keydown', this.hotKeyEventHandler);
```

Comment:
Can we just simply?
```
setupHotKeysEvents() {
  document.addEventListener('keydown', this. handleHotKeyEvents);
}
```

Reply:
Nope, because we need to bind 'this' to hotKeyEventHandler method. And this.hotKeyEventHandler.bind(this) also didn't work because it returns another function and it did not clear event listener in this case.